addons/website/views/snippets.xml

```diff
@@ -1462,8 +1462,12 @@
     <!-- Stretch section -->
     <div data-js="SectionStretch"
         data-selector="section"
-        data-target="> .container, > .container-fluid">
-        <we-checkbox string="Stretch" data-select-class="container|container-fluid" data-no-preview="true"/>
+        data-target="> .container, > .container-fluid, > .o_container_small">
+        <we-select string="Width">
```

Comment:
You can already add the `o_we_inline` class on this select as FP will want it for all selects in a future task.